docs/basics/DeepRLTutorial/TutorialDeepRL.rst

```diff
@@ -0,0 +1,605 @@
+Deep Reinforecement Learning in rlberry
+=======================================
+
+In this tutorial we will focus on Deep Reinforcement Learning with an
+**Actor-Advantage Critic** algorithm.
+
+Install, import and utilities
+-----------------------------
+
+.. code:: ipython3
+
+    from rlberry.envs import gym_make
+    from rlberry.manager import plot_writer_data, AgentManager, evaluate_agents
+    from rlberry.agents.torch import A2CAgent
+    from rlberry.agents.torch.utils.training import model_factory_from_env
+
+
+Reminder of the RL setting
+--------------------------
+
+As always we will consider a MDP
+:math:`M = (\mathcal{S}, \mathcal{A}, p, r, \gamma)` with:
+
+* :math:`\mathcal{S}` the state space,
+* :math:`\mathcal{A}` the action space,
+* :math:`p(x^\prime \mid x, a)` the transition probability,
+* :math:`r(x, a, x^\prime)` the reward of the transition :math:`(x, a, x^\prime)`,
+* :math:`\gamma \in [0,1)` is the discount factor.
+
+A policy :math:`\pi` is a mapping from the state space :math:`\mathcal{S}` to the probability of selecting each action.
+The action value function of a policy is the overall expected reward
+from a state action.
+:math:`Q^\pi(s, a) = \mathbb{E}_{\tau \sim \pi}\big[R(\tau) \mid s_0=s, a_0=a\big]`
+where :math:`\tau` is an episode
+:math:`(s_0, a_0, r_0, s_1, a_1, r_1, s_2, ..., s_T, a_T, r_T)` with the
+actions drawn from :math:`\pi(s)`; :math:`R(\tau)` is the random
+variable defined as the cumulative sum of the discounted reward.
+
+The goal is to maximize the agent’s reward.
+
+.. math::  J(\pi) = \mathbb{E}_{\tau \sim \pi}\big[R(\tau) \big]
+
+Gym Environment
+---------------
+
+In this tutorial we are going to use the `OpenAI’s Gym
+library <https://gym.openai.com/envs/>`__. This library provides a large
+number of environments to test RL algorithm.
+
+We will focus on the **CartPole-v1** environment, anyway, it might be
```

Comment:
Suggestion: "We will focus only on the **CartPole-v1** environment, although we recommend experimenting with other environments such as **Acrobot-v1** and **MountainCar-v0**."